uint32 size
float64 min_distance
geometry_msgs/Point initial
geometry_msgs/Point centroid
geometry_msgs/Point middle
geometry_msgs/Point initial_nbr
geometry_msgs/Point middle_nbr
geometry_msgs/Point centroid_nbr